packages/admin/resources/views/components/layouts/app/sidebar/index.blade.php

```diff
@@ -2,48 +2,74 @@
     @else
         x-cloak="-lg"
         x-bind:class="$store.sidebar.isOpen ? 'filament-sidebar-open translate-x-0' : '-translate-x-full lg:translate-x-0 rtl:lg:-translate-x-0 rtl:translate-x-full'"
     @endif
-
     @class([
         'filament-sidebar fixed inset-y-0 left-0 rtl:left-auto rtl:right-0 z-20 flex flex-col h-screen overflow-hidden shadow-2xl transition-all bg-white lg:border-r rtl:lg:border-r-0 rtl:lg:border-l w-[var(--sidebar-width)] lg:z-0',
         'lg:translate-x-0' => ! config('filament.layout.sidebar.is_collapsible_on_desktop'),
         'dark:bg-gray-800 dark:border-gray-700' => config('filament.dark_mode'),
     ])
 >
     <header @class([
-        'filament-sidebar-header border-b h-[4rem] shrink-0 px-6 flex items-center',
+        'filament-sidebar-header border-b h-[4rem] shrink-0 flex items-center justify-center',
         'dark:border-gray-700' => config('filament.dark_mode'),
     ])>
-        <a
-            href="{{ config('filament.home_url') }}"
-            class="block w-full"
-            @if (config('filament.layout.sidebar.is_collapsible_on_desktop'))
-                x-show="$store.sidebar.isOpen"
-                x-transition:enter="lg:transition delay-100"
-                x-transition:enter-start="opacity-0"
-                x-transition:enter-end="opacity-100"
-            @endif
-            data-turbo="false"
+        <div
+            x-cloak
+            @class([
+                'flex items-center jusify-center px-6 w-full',
+                'lg:px-4' => config('filament.layout.sidebar.is_collapsible_on_desktop') && (config('filament.layout.sidebar.collapsed_width') !== 0),
+            ])
+            x-show="$store.sidebar.isOpen || @js(! config('filament.layout.sidebar.is_collapsible_on_desktop')) || @js(config('filament.layout.sidebar.collapsed_width') === 0)"
         >
-            <x-filament::brand />
-        </a>
+            @if (config('filament.layout.sidebar.is_collapsible_on_desktop') && (config('filament.layout.sidebar.collapsed_width') !== 0))
+                <button
+                    type="button"
+                    class="filament-sidebar-collapse-button shrink-0 hidden lg:flex items-center justify-center w-10 h-10 text-primary-500 rounded-full hover:bg-gray-500/5 focus:bg-primary-500/10 focus:outline-none"
+                    x-on:click.stop="$store.sidebar.isOpen ? $store.sidebar.close() : $store.sidebar.open()"
+                    x-transition:enter="lg:transition delay-100"
+                    x-transition:enter-start="opacity-0"
+                    x-transition:enter-end="opacity-100"
+                    data-turbo="false"
+                >
+                    {{-- https://github.com/codeat3/blade-google-material-design-icons/blob/main/resources/svg/menu-open-r.svg --}}
```

Comment:
@danharrin we should really change this icon to the Heroicons chevron left before releasing this. Looks super inconsistent now.

Reply:
Okay will sort it

Reply:
Based on Heroicons v2

<img width="1440" alt="Screenshot 2022-09-17 at 19 25 29" src="https://user-images.githubusercontent.com/41773797/190866813-d38648f4-1d68-411a-b688-239ccbe0cc58.png">

<img width="1440" alt="Screenshot 2022-09-17 at 19 25 37" src="https://user-images.githubusercontent.com/41773797/190866808-3f54fcee-6772-48d4-a32b-dd86e0be6270.png">